apps/web-giddh/src/app/settings/settings.component.css

```diff
@@ -1,3 +1,18 @@
 .setting-container {
-    padding: 0 15px;
+    padding:0 15px 15px;
+    margin-top: 15px;
+}
+@media(max-width:767px){
+    .setting-sidebar-wrapper{
+        z-index: 999;
+        width: 100%;
+        height: 100%;
+        display: block;
+        background-color: rgba(0,0,0,0.5);
+        position: absolute;
+        top: 0; 
+    }
+    .ml-215{
```

Comment:
remove this css & use for this class ml-0

Reply:
Resolved